src/acquisition/covidcast/csv_to_database.py

```diff
@@ -127,18 +129,19 @@ def upload_archive(
     all_rows_valid = rows_list and all(r is not None for r in rows_list)
     if all_rows_valid:
       try:
-        result = database.insert_or_update_bulk(rows_list)
-        print(f"insert_or_update_bulk {filename} returned {result}")
+        modified_row_count = database.insert_or_update_bulk(rows_list)
+        print(f"insert_or_update_bulk {filename} returned {modified_row_count}")
         logger.info(
           "Inserted database rows",
-          row_count = result,
+          row_count = modified_row_count,
           source = source,
           signal = signal,
           geo_type = geo_type,
           time_value = time_value,
           issue = issue,
           lag = lag)
-        if result is None or result: # else would indicate zero rows inserted
+        if modified_row_count is None or modified_row_count: # else would indicate zero rows inserted
+          total_modified_row_count += (modified_row_count if modified_row_count else 0)
```

Comment:
you could do a `s/if modified_row_count else/or/` (but thats really just me playing code golf)